sormas-backend/src/main/java/de/symeda/sormas/backend/visit/VisitFacadeEjb.java

```diff
@@ -462,9 +465,25 @@ public List<VisitExportDto> getVisitsExportList(
 			if (resultList.size() > 0) {
 
 				Pseudonymizer pseudonymizer = Pseudonymizer.getDefault(userService::hasRight);
+				Map<Long, UserReference> visitUsers =
+					userService
+						.getUserReferencesByIds(
+							resultList.stream()
```

Comment:
Please do `.map(VisitExportDto:: getVisitUserId)` instead of `.map(c -> Arrays.asList(c.getVisitUserId())).flatMap(Collection::stream)`

There is no need to create a stream of lists with a single item than flatMap on it

Reply:
True